spec/support/controller_helpers.rb

```diff
@@ -0,0 +1,9 @@
+module ControllerHelpers
+  def stub_current_user(user)
+    allow(controller).to receive(:current_user).and_return(user)
+  end
+
+  def stub_authorize
+    allow(controller).to receive(:authorize!).and_return(true)
+  end
+end
```

Comment:
def sign_in
  #create your user and session here
end

Now you can reuse this helper to sign users in the test files